src/frontend/src/components/NERDraggableForm.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Typography, useTheme, IconButton } from '@mui/material';
+import NERFailButton from './NERFailButton';
+import NERSuccessButton from './NERSuccessButton';
+import Draggable from 'react-draggable';
+import CloseIcon from '@mui/icons-material/Close';
+import { ReactNode } from 'react';
+
+interface NERDraggableFormProps {
+  open: boolean;
+  title: string;
+  children?: ReactNode;
+  cancelText?: string;
```

Comment:
@RChandler234 do we have types for possible cancel / submit texts?

Reply:
Nope, since it literally just corresponds to a text displayed on the button, I don't think it needs a type

Reply:
i just meant in case we wanted to type it to get that crisp autocomplete

Reply:
like type cancelText = ‘cancel’ | ‘delete’ | ‘exit’ and then do the same thing for submit with like save, submit, create